### ISSUE TITLE
Fix Nav a11y (focus not obscured)

### ISSUE THREAD

**Issue by mark-wiemer** [mark-wiemer/hello-hello#44]
Right now, even when the Nav is closed, pressing `Tab` takes users from the nav button to the first link in the nav. That first link is completely hidden. Users have to press Tab many times to get past the closed nav and into the main content. See also #34.

We should make it such that the nav has no focusable elements while it's closed. This way, pressing Tab will skip from the nav button directly to the main content.
